src/strings/utf8.c

```diff
@@ -470,9 +470,7 @@ char * MVM_string_utf8_encode_substr(MVMThreadContext *tc,
         else {
             MVM_free(result);
             MVM_free(repl_bytes);
-            MVM_exception_throw_adhoc(tc,
-                "Error encoding UTF-8 string: could not encode codepoint %d",
-                cp);
+            MVM_utf8_throw_encoding_exception(tc, cp);
```

Comment:
`MVM_string_utf8_throw_encoding_exception` would follow the naming convention of other things in the file better.